src/client/components/app-header/app-header.vue

```diff
@@ -226,7 +226,7 @@ export default {
     box-shadow: none;
   }
 
-  .app-header__button--mobile {
+  .app-header .app-header__button--mobile {
```

Comment:
don't do this. stick with BEM. put an element around the button and set that to `display: none;`